tests/thirdparty/test_thirdparty.py

```diff
@@ -153,7 +141,9 @@ async def invalid_access_token(  # pylint: disable=unused-argument
 
 
 async def valid_access_token(  # pylint: disable=unused-argument
-    access_token: str, config: ProviderConfig, user_context: Optional[Dict[str, Any]]
+    access_token: str,
+    config: ProviderConfigForClient,
+    user_context: Optional[Dict[str, Any]],
 ):
     if access_token == "accesstoken":
```

Comment:
you should also make this function set some sort of variable and check for that variable value in the test - to make sure that this function is called.

Reply:
Done